bin/pnpm

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+# Set scripts-prepend-node-path=true to keep the version of node used to run pnpm on the PATH
+export npm_config_scripts_prepend_node_path=true
+
+# Set enable-pre-post-scripts=true to run pre/post scripts automatically in pnpm 6.0.0+
+export npm_config_enable_pre_post_scripts=true
```

Comment:
-2ing due to https://github.com/UrbanCompass/uc-frontend/pull/79351/files#r774235056

Reply:
I found the docs I was looking for here: https://github.com/UrbanCompass/uc-frontend/pull/79351/files#r774236979

This PR is fine. In the future it would be helpful if the implementation context was on the PR Itself